packages/jest-worker/src/workers/__tests__/processChild.test.js

```diff
@@ -241,8 +241,6 @@ it('returns results immediately when function is synchronous', () => {
 });
 
 it('returns results when it gets resolved if function is asynchronous', async () => {
-  jest.useRealTimers();
```

Comment:
Does nothing here. These errored as undefined at some point. Can be left in place for now.